services/basic_types.proto

```diff
@@ -95,12 +95,17 @@ message AccountID {
          * a protobuf Key message for any primitive key type. Currently only primitive key bytes are supported as an alias
          * (ThresholdKey, KeyList, ContractID, and delegatable_contract_id are not supported)
          *
+         * May also be the ethereum account 20-byte EVM address to be used initially in place of the public key bytes.
```

Comment:
Currently, this EVM address can only be the `shard.realm.num` format encoded as 20 bytes and not the hash form. If that is remaining true, then we should explicitly state that it only supports the zero-prefixed `shard-realm.num` format. If that is changing, then we should explicitly state both formats are supported now.

Reply:
Updated